site/admin/theatres/edit.php

```diff
@@ -0,0 +1,151 @@
+<?php
+include_once('init.php');
+PrintHeader('Edit Theatre', 'Edit Theatre');
+
+
+echo"<div class='ContactForm'>
+	<form action='' method='get'>
+	Which theatre do you want to edit?
+	<br /><br />";
+@TextBox('EditTheatre');
+echo"<br /><br />
+	<input type='submit' name='EditTheatreSubmitted' />
+	</form>
+	</div>";
+
+if(isset($_REQUEST['EditTheatreSubmitted'])){
+	if(!@$_REQUEST['EditTheatre']){
+		$FormErrors['EditTheatre'] = "Please enter a theatre to edit.";
+	}
+	if(sizeof($FormErrors) == 0){
+		$TheatreInfo = GetTheatre($_REQUEST['EditTheatre']);
+		//echo"".$TheatreInfo['Email']."";
+		//var_dump($TheatreInfo);
+		foreach($TheatreInfo as $Row){
```

Comment:
It seems weird to loop through an array of one. This loop shouldn't be necessary. The problem right now is that the `GetTheatre` function calls `fetchAll` instead of `fetch`